src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -216,9 +216,25 @@ public void Initialize()
             var path = Configuration.Core.DataDirectory.Combine("System");
             var storeAlertForLateRaise = new List<AlertRaised>();
 
-            var options = Configuration.Core.RunInMemory
-                ? StorageEnvironmentOptions.CreateMemoryOnly()
-                : StorageEnvironmentOptions.ForPath(path.FullPath);
+            StorageEnvironmentOptions options;
+            if (Configuration.Core.RunInMemory)
+            {
+                options = StorageEnvironmentOptions.CreateMemoryOnly();
+            }
+            else
+            {
+                options = StorageEnvironmentOptions.ForPath(path.FullPath);
+                var secretKey = Path.Combine(path.FullPath, "secret.key.encrypted");
+                if (File.Exists(secretKey))
```

Comment:
@ayende there is no error here... If we don't find the file we continue normally without encrypting the store.